### DISCUSSION TITLE
E77-400M22S, STM32WL55, J-Link, STM32Cube Programmer

### DISCUSSION THREAD

**Discussion by davefes** [micropython/micropython#13117] [STM32 / Pyboard]
I can not get a REPL up on this board using a FTDI programmer
on the LP_USART pins.

The manufacturer shows a programming setup using J-Link on the
SWDIO and SWCLK pins.  Can you put files on using J-Link?

Can I use STM32Cube Programmer to interact with Micropython
on the device?

Thanks.

**Comment by davefes** [comment]
Check your pin assignments VERY carefully!!

**Comment by projectgus** [comment]
Note this is a duplicate of https://github.com/orgs/micropython/discussions/13110
